modules/tests/test_ref_modules/input_stub_signal_helper.h

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2022-2023 Blueberry d.o.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+#include <coreobjects/unit_factory.h>
+#include <opendaq/context_factory.h>
+#include <opendaq/data_descriptor_factory.h>
+#include <opendaq/data_rule_factory.h>
+#include <opendaq/instance_factory.h>
+#include <opendaq/packet_factory.h>
+#include <opendaq/reader_utils.h>
+#include <opendaq/sample_type_traits.h>
+#include <opendaq/scheduler_factory.h>
+#include <opendaq/signal_factory.h>
+#include <testutils/testutils.h>
+#include <random>
+
+using namespace daq;
+
+struct InputStubSignal
```

Comment:
This isn't really a signal, but an entire test app setup. Considering that, I'd also add the "ref_fb_classifier" function block in the setup method, and rename the struct appropriately to "ClassifierTestHelper" or something similar.